lib/Container/Compiler/AggregateElasticsearchContentFieldMapperMapperPass.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\IbexaSearchExtra\Container\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Reference;
+use function array_keys;
+
+/**
+ * This compiler pass will register Content translation subdocument mappers.
+ *
+ * @see \Netgen\IbexaSearchExtra\Core\Search\Solr\SubdocumentMapper\ContentTranslationSubdocumentMapper
```

Comment:
This is c/p from another implementation, it should be updated.

Reply:
resolved in https://github.com/netgen/ibexa-search-extra/commit/497a329387786154c853232148c58bd86e3e230c